Automattic/wp-calypso
# Plan Pill Component: Migrate Styling

#### Changes proposed in this Pull Request

* Migrates the styling of the plan pill component added a few days ago, as part of #27515.

#### Testing instructions

Visit `/plans/site` and verify that the "Popular" icon is in line with a background colour, instead of the far right without styling. Basically, it should be identical. 

cc @jsnajdr, @blowery, @flootr
